lib/eyeshade/base_balance.rb

```diff
@@ -0,0 +1,51 @@
+module Eyeshade
+  class BaseBalance
+    attr_reader :rates,
+                :default_currency,
+                :amount_probi,
+                :amount_bat,
+                :amount_default_currency,
+                :fees_probi,
+                :fees_bat,
+                :fees_default_currency
+
+    # Account types
+    CHANNEL = "channel".freeze
```

Comment:
Let's instead do this at the top of the file. [Ruby 3.0 is going to make strings immutable by default.](https://wyeworks.com/blog/2015/12/1/immutable-strings-in-ruby-2-dot-3)
```ruby
# frozen_string_literal: true
```